prog/vnet/cert_nexus.rb

```diff
@@ -98,7 +98,11 @@ def before_run
 
   label def destroy
     # the reason is chosen as "cessationOfOperation"
-    acme_client.revoke(certificate: cert.cert, reason: REVOKE_REASON) if cert.cert
+    begin
```

Comment:
commit message:

>  In case Cert is already revoked, simply omit the error message 

...I mean, seems reasonable, but..why?  By error message, do you mean the stack trace? It's pretty different to go from "exception raised from the method" to "the exception is *not* raised". Seems more significant than a matter of error messages.

Reply:
Alright, that makes sense, I have added the exception and the stack trace into the log.